### DISCUSSION TITLE
Why a new compile don't delete old .gz files?

### DISCUSSION THREAD

**Discussion by weqopy** [nickjj/flask-static-digest#25] [Q&A]
Is this a feature or some special reason?  And it seems like no impact after I delete those files manually.

**Comment by nickjj** [comment]
The old `.gz` files are tied into the assets from the current or previous builds. Keeping around old files is a user decision. In most cases you'll probably want to delete them after deploying your new version but in some cases you may want to keep them, such as to not break old hotlinks (folks linking directly to one of your images, etc.).

If you want to delete them you can add that step into your deployment pipeline since this delete step can't happen during compile since you might be compiling your assets on a server that isn't where you host everything so the old files wouldn't be available.
